spec/features/user_contributes_new_councillor_spec.rb

```diff
@@ -18,6 +18,56 @@
       end
     end
 
+    scenario "on the contribution page" do
```

Comment:
I changed the description of the test and it made it clear that it is about the display logic. moved it to spec/views.